lib/posix/options/clock.c

```diff
@@ -43,6 +43,24 @@ int z_impl___posix_clock_get_base(clockid_t clock_id, struct timespec *base)
 			*base = rt_clock_base;
 		}
 		break;
+#ifdef CONFIG_SCHED_THREAD_USAGE_ANALYSIS
+	case CLOCK_THREAD_CPUTIME_ID:
```

Comment:
```suggestion
#ifdef CONFIG_POSIX_THREAD_CPUTIME
	case CLOCK_THREAD_CPUTIME_ID:
```

Reply:
It seems that `CLOCK_THREAD_CPUTIME_ID` is defined directly in `zephyr/posix/time.h`